test/cypress/integration/widgets/filter_spec.ts

```diff
@@ -18,9 +21,17 @@ describe('Filter', () => {
         cy.get('#dynamicDropdown1 > .label').should('have.length', 1);
 
         cy.get('#dynamicDropdown2').click();
```

Comment:
Can we use

```suggestion
        cy.get('#dynamicDropdown2').within(() => {
          cy.click();
          // other commands that do not start with #dynamicDropdown2
        });
```

to avoid repetition?